routes/userRoutes.js

```diff
@@ -27,10 +26,10 @@ userRouter.get('/me', userController.getUserProfile);
 userRouter.put(
   '/me',
   [
-    check('firstName').not().isEmpty(),
-    check('lastName').not().isEmpty(),
+    check('firstName').isString().notEmpty(),
+    check('lastName').isString().notEmpty(),
     check('email').normalizeEmail().isEmail(),
-    check('password').not().isEmpty(),
+    check('password').isString().notEmpty(),
     check('licensePlate').isLength({ min: 6, max: 6 }),
```

Comment:
Probably don't need to check for the` licensePlate `input (and maybe also the` password`) since updateUserProfile doesn't need it?